csharp/src/Google.Protobuf.Benchmarks/Benchmarks.cs

```diff
@@ -186,6 +186,23 @@ public sealed partial class BenchmarkDataset : pb::IMessage<BenchmarkDataset> {
       }
     }
 
+    #if SUPPORT_SPAN
```

Comment:
Make this the inverse. Allow people to opt out, rather than forcing them go through additional steps to opt in

`#if SUPPORT_SPAN` -> `#if !PROTOBUF_DISABLE_BUFFER_SERIALIZATION`